core/datastore/src/main/java/com/google/samples/apps/nowinandroid/core/datastore/NiaPreferencesDataSource.kt

```diff
@@ -34,6 +36,21 @@ class NiaPreferencesDataSource @Inject constructor(
                 bookmarkedNewsResources = it.bookmarkedNewsResourceIdsMap.keys,
                 followedTopics = it.followedTopicIdsMap.keys,
                 followedAuthors = it.followedAuthorIdsMap.keys,
+                themeBrand = when (it.themeBrand!!) {
```

Comment:
it.themeBrand == true is safer because if for some reason it's null, it will default to false. And below.

Reply:
Added a `null` branch to handle this.